tests/view/items/test_items.py

```diff
@@ -0,0 +1,75 @@
+""" Tests for the items.
+"""
+from http import HTTPStatus
+from datetime import datetime
+
+from tests.view.items.mock_items import ItemMock, ItemsMock
+from timeless.items.views import ItemListView
+
+
+def test_list(client):
+    """ Test list is okay """
+    ItemListView(
+        items = ItemsMock(
+            items = [
+                {
+                    "id": 1,
+                    "name": "Pancakes",
+                    "stock_date": datetime.utcnow,
+                    "comment": "Really delicious pancakes made from flour and eggs",
+                    "company_id": 1,
+                    "created_on": datetime.utcnow,
+                    "updated_on": datetime.utcnow,
+                    "employee_id": 1
+                },
+                {
+                    "id": 2,
+                    "name": "Dental Floss",
+                    "stock_date": datetime.utcnow,
+                    "comment": "Dental floss from Montana salesmen",
+                    "company_id": 1,
+                    "created_on": datetime.utcnow,
+                    "updated_on": datetime.utcnow,
+                    "employee_id": 1
+                },
+                {
+                    "id": 3,
+                    "name": "Power Cable",
+                    "stock_date": datetime.utcnow,
+                    "comment": "Power cable made of leather and heavy metal",
+                    "company_id": 1,
+                    "created_on": datetime.utcnow,
+                    "updated_on": datetime.utcnow,
+                    "employee_id": 1
+                },
+                {
+                    "id": 4,
+                    "name": "Brick Block",
+                    "stock_date": datetime.utcnow,
+                    "comment": "Red block made from bricks, sometimes hide coins",
+                    "company_id": 1,
+                    "created_on": datetime.utcnow,
+                    "updated_on": datetime.utcnow,
+                    "employee_id": 1
+                }
+            ]
+        ),
+        item = ItemMock
+    )
+    assert client.get("/items").status_code == HTTPStatus.OK
```

Comment:
@paulodamaso Why do you set up `ItemListView ` if you do not use it?

Reply:
@vladarefiev According to the logic of these `GenericViews` I have to instantiate `ItemListView` so it register our urls and answer the calls to a given path. As far as I understood all these `GenericViews` we are trying to encapsulater all this logic making easier to create new views for the rest of our model objects and making it testable. So I we shouldn't need to pass this `ItemListView` to anywhere because it works like a service or something, am I right?

Reply:
@paulodamaso  No it doesn't work in this way and it's easy to check, just check a data in response.

Pls just do it in way which we are already use:
tests example https://github.com/timelesslounge/timelessis/blob/master/tests/integration/it_reservation_settings_test.py#L16
list view example https://github.com/timelesslounge/timelessis/blob/master/timeless/reservations/views.py#L19